README.md

```diff
@@ -0,0 +1,330 @@
+File fileWithClassifier = Maven.resolver().resolve("G:A:test-jar:tests:V").withTransitivity().asFile();
+```
+
+#### Returning Resolved Artifacts
+
+ShrinkWrap Resolvers provides various options to retrieve resolved artifacts. You can obtain them as files, streams, URLs, or specific archive types like JavaArchive, WebArchive, or EnterpriseArchive. For example:
+- Get as a URL: `Maven.resolver().resolve("G:A:V").withTransitivity().as(URL.class)`
+- Get as a Java archive: `Maven.resolver().resolve("G:A:V").withTransitivity().as(JavaArchive.class)`
+
+
+#### Working with Artifact Metadata
+```java
+MavenResolvedArtifact artifact = Maven.resolver().resolve("G:A:war:V").withoutTransitivity()
+  .asSingle(MavenResolvedArtifact.class);
+```
+
+You can still retrieve resolved artifact from `MavenResolvedArtifact`:
+```java
+File file = artifact.asFile();
+```
+
+#### Working with artifact coordinates
+```java
+MavenCoordinate[] coordinates = Maven.resolver().resolve("G:A:V")
+   .withTransitivity().as(MavenCoordinate.class);
+```
+
+#### Resolution of artifacts as collection
+```java
+List<File> files = Maven.resolver().resolve("G:A:V")
+    .withTransitivity().asList(File.class);
+```
+
+#### Exclude specific dependencies while resolving an artifact
+```java
+File file = Maven.resolver().addDependencies(
+    MavenDependencies.createDependency("G:A:V", ScopeType.COMPILE, false,
+      MavenDependencies.createExclusion("G:B"))).resolve().withTransitivity().asFile();
+```
+
+#### Control Resolution Results with Strategy
+```java
+File file = Maven.resolver().resolve("G:A:V").using(new RejectDependenciesStrategy(false, "G:B")).asFile();
+```
+
+#### ShrinkWrap Resolver Strategies
+
+In ShrinkWrap Resolver, strategies allow you to modify the dependency graph and change the default behavior for resolving transitive dependencies.
+
+By default, Maven and ShrinkWrap Resolver do not resolve dependencies in provided and test scope, and skip optional dependencies.
+
+#### Available Strategies:
+
+- **AcceptAllStrategy (Equals TransitiveStrategy):**
+  Accepts all dependencies of artifacts.
+
+- **AcceptScopesStrategy:**
+  Accepts only dependencies with defined scope types.
+
+- **CombinedStrategy:**
+  Combines multiple strategies with a logical AND operation.
+
+- **NonTransitiveStrategy:**
+  Rejects all transitive dependencies; only directly specified dependencies are accepted.
+
+- **RejectDependenciesStrategy:**
+  Rejects dependencies defined by G:A. By default, it is transitive, meaning all dependencies originating at G:A are removed. To change this behavior to reject defined dependencies but keep their descendants, use `RejectDependenciesStrategy(false, "G:A")`.
+
+- **TransitiveStrategy (Equals AcceptAllStrategy):**
+  Accepts all dependencies of artifacts.
+
+
+#### Control Sources of Resolution
+
+ShrinkWrap Resolvers allows you to specify artifact resolution sources. By default, it uses the classpath (Maven Reactor) and Maven Central repository.
```

Comment:
I believe the `workOffline` and `useLegacyLocalRepo` should also be explained since we're already explaining the other two